packages/typespec-ts/src/utils/clientUtils.ts

```diff
@@ -0,0 +1,18 @@
+import {
+  SdkClient,
+  SdkContext
+} from "@azure-tools/typespec-client-generator-core";
+import { listServices } from "@typespec/compiler";
+
+export function getRLCClients(dpgContext: SdkContext): SdkClient[] {
```

Comment:
I like how simple this is now.

Reply:
me too, sorry for any confusions I may have caused. I thought the boundary of client.tsp == the boundary of @client decorator.